grip_api/src/grip_api/config_widgets/plain_editor_widgets.py

```diff
@@ -65,7 +65,7 @@ def create_editor(self):
         """
             Initialize the code editor. Will be used by the derivated classes
         """
-        self.code_editor = GenericCodeEditor()
+        pass
```

Comment:
This will be modified when the corresponding file is refactored. This change does not break anything and ensures compatibility with the changes of this PR

Reply:
True dat, thanks!

Reply:
This will be done when the PR corresponding to this file will be done